### PR TITLE
Add proxy and remove tracking id

### PR DESCRIPTION
**Pull request by alex-tpom6oh**
Applying changes from https://github.com/avohq/ios-avo-inspector/pull/35

### CODE REVIEW

File: Sources/AvoInspector/AvoInspector.m [hunk 1]
```diff
@@ -113,6 +113,11 @@ -(instancetype) initWithApiKey: (NSString *) apiKey env: (AvoInspectorEnv) env {
     return self;
 }
 
+-(instancetype) initWithApiKey: (NSString *) apiKey env: (AvoInspectorEnv) env {
+    self = [self initWithApiKey:apiKey env:env proxyEndpoint:@"https://api.avo.app/inspector/v1/track"];
```
Comment (logason) [inline Sources/AvoInspector/AvoInspector.m:117]: Can we add a task to make sure we update docs along with this change?

File: Sources/AvoInspector/AvoNetworkCallsHandler.m [hunk 2]
```diff
@@ -128,7 +130,6 @@ - (NSMutableDictionary *) createBaseCallBody {
     [body setValue:[AvoNetworkCallsHandler formatTypeToString:self.env] forKey:@"env"];
     [body setValue:@"ios" forKey:@"libPlatform"];
     [body setValue:[[NSUUID UUID] UUIDString] forKey:@"messageId"];
-    [body setValue:[[AvoInstallationId new] getInstallationId] forKey:@"trackingId"];
```
Comment (logason) [inline Sources/AvoInspector/AvoNetworkCallsHandler.m:131]: Should we also stop generating and storing the trackingId in the SDK?

**Review by logason — APPROVED** [review]
LGTM!

File: Sources/AvoInspector/AvoInspector.m (hunk 1 above)
Comment (alex-tpom6oh) [reply Sources/AvoInspector/AvoInspector.m:117]: Yeah, good catch

**Pull request merged**